src/core/EdFi.DataManagementService.Core.Tests.Unit/Middleware/CoerceFromStringsMiddlewareTests.cs

```diff
@@ -143,7 +143,7 @@ public void It_coerces_booleans()
                 foreach (string path in _context.ResourceSchema.BooleanJsonPaths.Select(s => s.Value))
                 {
                     _context.ParsedBody.SelectNodeFromPath(path, NullLogger.Instance)!.AsValue().GetValueKind().Should()
-                        .BeOneOf([JsonValueKind.True, JsonValueKind.False]);
+                        .BeOneOf(JsonValueKind.True, JsonValueKind.False);
```

Comment:
`BeOneOf` accepts a `param` array, which means we don't need to _declare_ an array.